packages/react/src/VideoPlayer/VideoPlayer.tsx

```diff
@@ -0,0 +1,213 @@
+  useEffect(() => {
+    if (videoRef.current) {
+      for (let i = 0; i < videoRef.current.textTracks.length; i++) {
+        videoRef.current.textTracks[0].mode = 'hidden'
+      }
+    }
+  }, [videoRef])
+
+  /* > Set breakpoint classes                                                   */
+
+  useEffect(() => {
+    const handleResize = () => {
+      const breakpoint = videoWrapperRef.current?.getBoundingClientRect().width
+      if (breakpoint && breakpoint < 650) {
+        videoWrapperRef.current.classList.add(styles.VideoPlayer__breakpointSmall)
+      } else {
+        videoWrapperRef.current?.classList.remove(styles.VideoPlayer__breakpointSmall)
+      }
+    }
+
+    const resizeObserver = new ResizeObserver(_ => {
+      if (videoWrapperRef.current) {
+        handleResize()
+      }
+    })
+
+    handleResize()
+
+    const currentRef = videoWrapperRef.current
+
+    resizeObserver.observe(videoWrapperRef.current as Element)
+    return () => {
+      resizeObserver.unobserve(currentRef as Element)
+    }
+  }, [videoWrapperRef, playing])
+
+  return (
+    <div
+      className={clsx(
+        styles.VideoPlayer__container,
+        styles.VideoPlayer__overlays,
+        !playing && styles.VideoPlayer__showOverlays,
+      )}
+      ref={videoWrapperRef}
+    >
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <video
+        ref={videoRef}
+        title={title}
+        controls={false}
+        poster={poster}
+        className={clsx(styles.VideoPlayer, className)}
+        onPlay={handleOnPlay}
+        onPause={handleOnPause}
+        onPlaying={handleOnPlaying}
+        onLoadedMetadata={handleOnLoadedMetadata}
+        onTimeUpdate={handleOnTimeUpdate}
+        {...rest}
+      >
+        {children}
+      </video>
+      {title && (
+        <div className={styles.VideoPlayer__title}>
+          {branding && <MarkGithubIcon size={48} />}
+          <Text size="500" weight="medium" className={styles.VideoPlayer__controlTextColor}>
+            {title}
+          </Text>
+        </div>
+      )}
+      <button className={styles.VideoPlayer__playButton} onClick={handleVideoPlayback}>
+        {!playing && (
+          <span className={styles.VideoPlayer__playButtonInner}>
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
+              style={{marginLeft: 5}}
+              aria-hidden="true"
+            >
+              <path
+                d="M1.56489 23.8112C0.884817 24.2389 1.07491e-06 23.7501 1.0398e-06 22.9467L8.28679e-08 1.05473C4.73246e-08 0.241596 0.904067 -0.245404 1.58307 0.201969L18.5829 11.4026C19.2032 11.8113 19.1935 12.7244 18.5647 13.1198L1.56489 23.8112Z"
+                fill="currentColor"
+              />
+            </svg>
+          </span>
+        )}
+        <span className={styles.VideoPlayer__srOnly}>Play</span>
```

Comment:
we have a visually-hidden util class, can we use that instead here?

```suggestion
        <span className="visually-hidden">Play</span>
```

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5